templates/documents/upload_files_modal.html

```diff
@@ -11,7 +11,7 @@
             class="file-input file-input-bordered w-full max-w-xs"
             accept=".txt,.pdf,.jpg,.png"/>
 
-        <select class="select select-bordered w-full" name="collection_name">
+        <select id="upload-collections-select" name="collection_names[]" multiple placeholder="Select collections" autocomplete="off">
```

Comment:
nit: we don't use this field naming convention anywhere else
```suggestion
        <select id="upload-collections-select" name="collection_names" multiple placeholder="Select collections" autocomplete="off">
```